core/store/src/db.rs

```diff
@@ -882,8 +915,11 @@ impl RocksDB {
         RocksDBOptions::default().read_only(path)
     }
 
-    pub fn new<P: AsRef<std::path::Path>>(path: P) -> Result<Self, DBError> {
-        RocksDBOptions::default().read_write(path)
+    pub fn new<P: AsRef<std::path::Path>>(
+        path: P,
+        enable_statistics: bool,
```

Comment:
I’d probably keep `new` as a method which creates db with default options, and let the callers use RocksDBOptions API explicitly if they want more control. 

It kinda doesn’t make sense to have both options builder and a bunch of parameters to new

Reply:
Done